static/js/util/profile_edit.js

```diff
@@ -282,9 +289,5 @@ export function saveProfileStep(validator: Validator|UIValidator, isLastStep: bo
   let clone = Object.assign({}, profile, {
     filled_out: profile.filled_out || isLastStep
   });
-  if (isLastStep) {
-    // user has also seen email consent message at this point
-    clone.email_optin = true;
-  }
```

Comment:
I'm not sure about this deletion - we probably do want to set 'email_option' to true unless a user opts out, but we only want to do so if we're saving and the user hasn't yet completed the profile onboarding process.

maybe we want something like

```
if (isLastStep && !profile.filled_out) {
  ...
}
```

so that we only set the default value when the user is saving for the first time (i.e. when `filled_out` hasn't yet be set to `true`).